core/src/main/scala/kafka/tools/MirrorMaker.scala

```diff
@@ -83,6 +89,11 @@ object MirrorMaker extends Logging with KafkaMetricsGroup {
   def main(args: Array[String]) {
 
     info("Starting mirror maker")
+    println("WARNING: The default partition assignment strategy of mirror maker will change from 'range' to 'roundrobin' " +
```

Comment:
@hachikuji Yes, I understand that MM works as a consumer group consuming from a number of given topics. So my confusion seems to be on trivial matters. Am I correct to assume that once MM starts we can easily stop some of the consumers and restart them? I now feel that this is what you meant by MM processes (consumers rather than the MM command). Thanks :)